Lesson35/src/modules/user/LogInForm/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import './styles.css';
+
+const registeredUsers = [
+  { username: "johndoe01", password: "password123" },
+  { username: "janedoe02", password: "password456" },
+];
+
+export const LogInForm = ({ handleLogin }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const handleSubmit = (e) => {
```

Comment:
When form is submitted, you need to dispatch login action if the user data match data in the registered users list.